src/Algebra/GCD.agda

```diff
@@ -0,0 +1,62 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Definitions of irreducibility, primality, coprimality, GCD.
+------------------------------------------------------------------------
+
+-- You're unlikely to want to use this module directly. Instead you
+-- probably want to be importing the appropriate module from
+-- `Algebra.Properties.CancellativeCommutativeSemiring`.
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Algebra.Core using (Op₂)
+import Algebra.Divisibility
+open import Data.Product using (_×_)
+open import Data.Sum using (_⊎_)
+open import Function using (flip)
+open import Level using (_⊔_)
+open import Relation.Binary using (Rel; Symmetric)
+open import Relation.Nullary using (¬_)
+open import Relation.Unary using (Pred)
+
+module Algebra.GCD
+  {β ℓ} {A : Set β} (_≈_ : Rel A ℓ) (0# 1# : A) (_*_ : Op₂ A)
```

Comment:
We also usually put the binary operation before the specialised elements in the telescope.

Reply:
Done.